CMakeLists.txt

```diff
@@ -102,22 +106,44 @@ if(TARGET_PLATFORM STREQUAL "linux")
 endif()
 
 # Default values
-set(BEEROCKS_WLAN1_IFACE "wlan0")
-set(BEEROCKS_WLAN2_IFACE "wlan2")
-set(BEEROCKS_WLAN3_IFACE "wlan4")
-set(BEEROCKS_BRIDGE_IFACE "br-lan")
+if(NOT DEFINED BEEROCKS_WLAN1_IFACE)
+    set(BEEROCKS_WLAN1_IFACE "wlan0")
+endif()
+if(NOT DEFINED BEEROCKS_WLAN2_IFACE)
+    set(BEEROCKS_WLAN2_IFACE "wlan2")
+endif()
+if(NOT DEFINED BEEROCKS_WLAN3_IFACE)
+    set(BEEROCKS_WLAN3_IFACE "wlan4")
+endif()
```

Comment:
This one is actually not used, please remove it instead.

Reply:
Should i also remove next code from `agent/config/beerocks_agent.conf.in`?
```

[agent2]
radio_identifier=00:00:00:00:00:02
hostap_iface_type=WIFI_INTEL
hostap_ant_gain=0
enable_repeater_mode=0
```

Reply:
No, we will add a 3-radio platform at some point so that will still be useful. However, we'll probably won't have a docker test for it so the CMake option shouldn't be needed.